pkg/controller/jobs/jobset/jobset_controller.go

```diff
@@ -182,6 +184,31 @@ func (j *JobSet) PodsReady() bool {
 	return replicas == readyReplicas
 }
 
+func (j *JobSet) ReclaimablePods() []kueue.ReclaimablePod {
+	if len(j.Status.ReplicatedJobsStatus) == 0 {
+		return nil
+	}
+
+	ret := make([]kueue.ReclaimablePod, 0, len(j.Spec.ReplicatedJobs))
+	specs := slices.ToRefMap(j.Spec.ReplicatedJobs, func(js *jobsetapi.ReplicatedJob) string { return js.Name })
+	statuses := slices.ToRefMap(j.Status.ReplicatedJobsStatus, func(js *jobsetapi.ReplicatedJobStatus) string { return js.Name })
+
+	for name, spec := range specs {
+		if status, found := statuses[name]; found && status.Succeeded > 0 {
+			pc := podsCount(spec)
+			totalCount := pointer.Int32Deref(spec.Template.Spec.Completions, pointer.Int32Deref(spec.Template.Spec.Parallelism, 1)) * int32(spec.Replicas)
+			remaining := totalCount - status.Succeeded
```

Comment:
is `status.Succeeded` the number of successful pods across the replicated jobs?

Reply:
apparently no, reworked